chag

```diff
@@ -173,7 +214,7 @@ latest() {
 
 # Get the line number of a specific changelog entry.
 find_entry() {
-  result=$(entries_and_lines | grep -E ":${1//./\\.}$" | cut -d ':' -f 1)
+  result=$(entries_and_lines | grep "${1//./\\.}" | cut -d ':' -f 1)
```

Comment:
One thing that could be improved in the next command is that it could by default commit the modifications to the changelog file by simply executing `git add $changelogfile && git commit`.